examples/run_largecyc_CPMG.py

```diff
@@ -191,7 +191,8 @@
 data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis(
     "ph1", ph1 / 4
 ).setaxis("ph2", ph2 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_prop("postproc_type", "spincore_diffph_SE_v2")
```

Comment:
Can you actually replace SE with cpmg here? I did something like this at some point, but we have had clarifying conversations about the double use of this post-processing type property.